src/qt/utilitydialog.cpp

```diff
@@ -58,7 +58,8 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
         ui->helpMessage->setVisible(false);
     } else {
         setWindowTitle(tr("Command-line options"));
-        QString header = "Usage:  bitcoin-qt [command-line options]                     \n";
+        QString header = "Usage:  bitcoin-qt [command-line options] [BIP21 URI]                     \n\n"
```

Comment:
Probably better to just leave it as `[URI]` here, and if BIP21 is mentioned, only do so in the explanation below.

Reply:
Why not the [complete syntax representation](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#user-content-ABNF_grammar) of the BIP21 command?
```
 bitcoinurn     = "bitcoin:" bitcoinaddress [ "?" bitcoinparams ]
 bitcoinaddress = *base58
 bitcoinparams  = bitcoinparam [ "&" bitcoinparams ]
 bitcoinparam   = [ amountparam / labelparam / messageparam / otherparam / reqparam ]
 amountparam    = "amount=" *digit [ "." *digit ]
 labelparam     = "label=" *qchar
 messageparam   = "message=" *qchar
 otherparam     = qchar *qchar [ "=" *qchar ]
 reqparam       = "req-" qchar *qchar [ "=" *qchar ]

```

Reply:
@pablomartin4btc I think this is not the place to explain BIP21, and I'll follow @luke-jr 's advice on this one.